elements/lisk-p2p/src/peer_directory/new_list.ts

```diff
@@ -70,34 +67,28 @@ export class NewList extends PeerList {
 		// Second eviction strategy: Default eviction based on base class
 		return this.evictRandomlyFromBucket(bucketId);
 	}
+
 	// Evict a peer when a bucket is full based on the time of residence in a peerlist
-	private _evictionBasedOnTimeInBucket(
+	private _evictPeerBasedOnTimeInBucket(
```

Comment:
isn't `Peer` kind of redundant...? I think only thing we can evict is `Peer` 😉

Reply:
`_evictPeerBasedOnTimeInBucket` was chosen to be consistent with the `evictPeer` method.